docker/Dockerfile

```diff
@@ -36,7 +36,7 @@ ENTRYPOINT [ "/redis_exporter" ]
 #
 # Alpine release container
 #
-FROM --platform=linux/$GOARCH alpine:3.20.0 as alpine
+FROM --platform=linux/$GOARCH alpine:3.20.1 as alpine
```

Comment:
idea: let's make this 3.20 so it pulls in the most current minor version whenever we cut a release.